src/main/scala/com/softwaremill/react/kafka/KafkaActorSubscriber.scala

```diff
@@ -23,17 +22,12 @@ private[kafka] class KafkaActorSubscriber[T](
       processElement(element.asInstanceOf[T])
     case ActorSubscriberMessage.OnError(ex) =>
       handleError(ex)
-    case ActorSubscriberMessage.OnComplete =>
+    case ActorSubscriberMessage.OnComplete | "Stop" =>
```

Comment:
That's not an issue of Akka - it's "by design" according to reactive streams specs. `OnComplete` is meant to be an internal message of Akka Streams. When it's sent to a subscriber, one can't send any more elements (and onError will not be called). This is part of the specs, so I needed something custom to be able to close internal resource in order to simulate sending an element when Kafka is not available.